### PR TITLE
Expand VPC Subnet resource to meet the API

### PR DESCRIPTION
**Pull request by annakhm**

### CODE REVIEW

File: nsxt/resource_nsxt_vpc_subnet.go [hunk 1]
```diff
@@ -319,6 +388,152 @@ var vpcSubnetSchema = map[string]*metadata.ExtendedSchema{
 							ReflectType:  reflect.TypeOf(model.StaticPoolConfig{}),
 						},
 					},
+					"dhcp_relay_config": {
+						Schema: schema.Schema{
+							Type:     schema.TypeList,
+							MaxItems: 1,
+							Elem: &metadata.ExtendedResource{
+								Schema: map[string]*metadata.ExtendedSchema{
+									"server_addresses": {
```
Comment (ksamoray) [inline nsxt/resource_nsxt_vpc_subnet.go:397]: Validate element with `validateSingleIP`

**Review by ksamoray — COMMENTED** [review]
Minor stuff, otherwise cool

**Review by salv-orlando — COMMENTED** [review]
LGTM once you review Kobi's comments.

**Review by salv-orlando — APPROVED** [review]
LGTM

**Review by ksamoray — APPROVED** [review]

**Pull request merged**